packages/calypso-e2e/src/lib/pages/editor-page.ts

```diff
@@ -49,7 +49,7 @@ const selectors = {
  */
 export class EditorPage {
 	private page: Page;
-	private editor: EditorComponent;
+	public editor: EditorComponent;
```

Comment:
Everything else uses accessors rather than public properties, why change that?

Reply:
Indeed! c6273a390b